regression/invariants/invariant-failure/test.desc

```diff
@@ -7,6 +7,8 @@ string
 Test invariant failure
 Invariant check failed
 ^(Backtrace)|(Backtraces not supported)$
+^Condition: false$
+^Reason: Test invariant failure$
```

Comment:
I'd strongly prefer adding another test for this rather than piggy-backing off an existing one.

Reply:
I am fairly certain this is the place for this to be added. The first test doesn't seem to do anything fairly important otherwise, and this is just adding some extra lines that should be there and are standard for every invariant failure.